docs/EN_US/DynamicESDL/DESDL-Mods/ESDLping_min_ver.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect1 id="ESDL_Attributes_ping_min_ver">
+  <title>ping_min_ver</title>
+
+  <para>Dynamic ESDL services automatically add a ping method for testing and
```

Comment:
it's not necessarily for testing/development, it's more for operations/monitoring.